_includes/specials/policy/en/03-bidding-for-block-space.md

```diff
@@ -0,0 +1,105 @@
+<!--
+  300 to 1000 words
+  put title in main newsletter
+  put links in this file
+  for any subheads use h3 (i.e., ###)
+  illustrations welcome (max width 800px)
+  if uncertain about anything, just do what seems best and harding will edit
+-->
+
+Last week we mentioned that transactions pay fees for the used
+blockspace rather than the transferred amount, and established that
+miners optimize their transaction selection to maximize collected fees.
+It follows that only those transactions get confirmed that reside in the
```

Comment:
maybe

It follows that only transactions that reside in the top of the mempool get confirmed when a block is found.